CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # `dev-master`
 
+* #414 Remove useless blank line (@mageekguy)
```

Comment:
I agree although I'm at the origin of the change.

I think i will write a little whitepaper on "how to format and fill in the
changelog"

Le samedi 3 janvier 2015, Frédéric Hardy <notifications@github.com> a
écrit :

> In CHANGELOG.md
> <https://github.com/atoum/atoum/pull/416#discussion-diff-22431577>:
>
> > @@ -1,5 +1,6 @@
> >  # `dev-master`
> >
> > +* #414 Remove useless blank line (@mageekguy)
>
> HUGE +1.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/atoum/atoum/pull/416/files#r22431577>.
>


-- 
Mikael Randy
Coordinateur AFUP Lyon
Administrateur PHPFrance